photon-lib/build.gradle

```diff
@@ -112,6 +116,20 @@ model {
                     srcDirs "src/generate/native/include"
                 }
             }
+            it.binaries.all {
+                it.tasks.withType(CppCompile) {
+                    it.dependsOn ":photon-targeting:generateProto"
+                }
+            }
+        }
+    }
+    testSuites {
+        all {
+            it.binaries.all {
+                it.tasks.withType(CppCompile) {
+                    it.dependsOn ":photon-targeting:generateProto"
+                }
+            }
```

Comment:
Why does this need to be done? photon-lib shouldn't have a direct dependency on photon-targeting's generateProto task. This falls outside of my gradle knowledge so we may be doing something wrong/inefficently here